pyscf/adc/dfadc.py

```diff
@@ -15,6 +15,47 @@
 
 import numpy as np
 
+def  get_ovvv_df(myadc, Lov, Lvv, p, chnk_size):
+
+    nocc = myadc._nocc
+    nvir = myadc._nvir
+    naux = myadc.with_df.get_naoaux()
+
+    Lvv = Lvv.reshape(naux,nvir*nvir)
+    Lov = Lov.reshape(naux,nocc,nvir)
+
+    if chnk_size < nocc:
+        Lov_temp = np.ascontiguousarray(Lov.transpose(1,2,0)[p:p+chnk_size].reshape(-1,naux))
+    else :
+        Lov_temp = np.ascontiguousarray(Lov.transpose(1,2,0).reshape(-1,naux))
+
+    ovvv = np.dot(Lov_temp, Lvv)
+    ovvv = ovvv.reshape(-1, nvir, nvir, nvir)
+    return ovvv    
+
+
+def  get_ovvv_a_df(myadc, Lov, Lvv, p, chnk_size):
+
+    norb_1 = Lov.shape[1]
+    norb_2 = Lov.shape[2]
+    norb_3 = Lvv.shape[1]
+    naux = myadc.with_df.get_naoaux()
```

Comment:
You might want to use more descriptive variable names. It seems to me that `norb_1 = Nocc`, `norb_2 = Nvirt`, and `norb_3 = Nvirt`

Reply:
You actually use better variable names above in `get_ovvv_df`

Reply:
This function is used in the unrestricted code to calculate ovvv integrals with different spin cases. So, norb_1 can be nocc_alpha/ nocc_beta. Same for the virtual orbitals as well. This is why I have used generic variable names.

Reply:
If `norb_1` is always `nocc_alpha` or `nocc_beta`, why not just name it `nocc`? Same for the virtual orbitals.

Reply:
Yeah, norb_1 can be named nocc. But norb_2 and norb_3 will be dependent on Lov and Lvv, respectively. In that case, norb_2 can be named nvir_1, and norb_3 can be named nvir_2. I can make these changes.